data/net/minecraft/world/level/levelgen/Column.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/world/level/levelgen/Column
+	CLASS Ray
+		METHOD <init> (IZ)V
+			ARG 1 edge
+			ARG 2 pointingUp
+	CLASS Range
+		METHOD <init> (II)V
+			ARG 1 floor
+			ARG 2 ceiling
```

Comment:
This is hard for me to say without looking at the code, but these seem odd choices - would they be better as `min` / `max`, or if clarification is necessary, e.g. `minInclusive`, `maxExclusive`

Reply:
These are field names